openfasoc/generators/cryo-gen/tools/parse_rpt.py

```diff
@@ -22,3 +25,13 @@
     raise ValueError("LVS failed!")
 else:
     print("LVS is clean!")
+
+json_filename = "test.json"
+
+if os.path.exists(json_filename):
+    if check_gen_files():
```

Comment:
I think it would be good to pass the `json_file_name` as an argument to `check_gen_files()` so that you can avoid changing this filename at two places (at the generator side as well as inside the `common/check_gen_files.py`.